src/scraping/pilkanoznapl/parser.py

```diff
@@ -131,3 +134,9 @@ def format_data():
     df = pd.read_csv(pth, sep='|')
     df = assign_id_to_articles(df)
     df.to_csv(pth, sep='|', index=False)
+
+def parse_date(date_string: str) -> datetime:
```

Comment:
To pewnie by się dało zastąpić f-cją [strptime()](https://www.journaldev.com/23365/python-string-to-datetime-strptime)